src/openforms/tests/e2e/input_validation_base.py

```diff
@@ -133,7 +140,7 @@ def _assertBackendValidation(self, form: Form, key: str, value: JSONValue):
         # step data validation is run *if* a value is provided - it ignores empty data
         # for fields that are required. So we accept an HTTP 400, or if a 200/201 is
         # returned, then we apply additional checks with the _complete endpoint.
-        assert response.status_code in [201, 200, 400]
+        assert response.status_code in (201, 200, 400)
```

Comment:
So that the match statement correctly exhausts all possible cases

Reply:
TIL

Reply:
That's a pyright only thing by the way, mypy will still understand it as a tuple of ints